web/app.py

```diff
@@ -114,6 +120,8 @@ def crawl():
         sleep(duration.seconds)
         message = f"timeout; adding backoff for {domain}"
         return {"error": {"message": message}}, 429
+    finally:
+        sleep(retry_duration)
```

Comment:
Perhaps it would make more sense to add this to `domain_backoffs`, instead of delaying the individual in-flight request?